exist-core/src/test/xquery/subsequence.xq

```diff
@@ -30,3 +30,26 @@ declare
 function ss:pipeline1() {
     ("x", "y") => subsequence(1, 1) => matches("x")
 };
+
+declare
+    %test:args(1,1)
+    %test:assertEmpty
+    %test:args(3,1)
+    %test:assertEmpty
+    %test:args(5,1)
+    %test:assertEmpty
+    %test:args(1,2)
+    %test:assertEquals(2)
+    %test:args(3,2)
+    %test:assertEquals(4)
+    %test:args(5,2)
+    %test:assertEmpty(4)
+    %test:args(1,3)
+    %test:assertEquals(2,3)
+    %test:args(3,3)
+    %test:assertEquals(4,5)
+    %test:args(1,5)
+    %test:assertEquals(2,3,4,5)
+function ss:tail($start, $length) {
+    subsequence((1 to 5), $start, $length) => tail()
```

Comment:
Can you remove the unnecessary arrow operator please? (for the reasons previously discussed about the ability to debug such constructs from Java).

Reply:
Do you really think this warrants to reject this PR?

Reply:
The test above has an arrow operator.
I also do used _this_ test to debug _and_ fix the issue. It is certainly possible.

Reply:
> his warrants to reject this PR?

I am not rejecting the PR, I am asking for a very small change. Surely the purpose of the peer-review process is exactly for for these matters!

> The existing XQSuite test case above the newly added one has an arrow operator as well.

I don't see what bearing that has on your changes. If the existing code was perfect, then we would never need to make changes to eXist-db! When we are making changes, these days we follow the peer-review process, which hopefully leads to agreed improvements to the code base.